readme.md

```diff
@@ -153,256 +184,178 @@ g1-b1-i1 g2-b3-i1     1           PO                0.4999          0.2467
-    launcher.py vcf --vcf-file /media/ref/input.vcf.gz --directory /media/pipeline_data/imputed_data \
-    --real-run
-    
-    # now we can find relatives
-    docker run --rm --privileged -it -v /media:/media -v /etc/localtime:/etc/localtime:ro genx_relatives:latest \
-    launcher.py find --directory /media/pipeline_data/imputed_data \
-    --until merge_imputation_filter --real-run
-    ```
-    Then, one can grab file `/media/pipeline_data/imputed_data/vcf/merged_imputed.vcf.gz`
+```bash
+docker run --rm --privileged -it -v /media:/media -v /etc/localtime:/etc/localtime:ro \
+    genx_relatives:latest launcher.py simulate --flow ibis-king --ref-directory /media/ref \
+        --directory /media/data --sim-params-file params/relatives_average.def \
+        --sim-samples-file ceph_unrelated_all.tsv --assembly hg37 --ibis-seg-len 5 \
+        --ibis-min-snp 400 --zero-seg-count 0.1 --zero-seg-len 5 --alpha 0.01 --real-run
+```
 
+### Precision / Recall Analysis
 
-### Evaluation on Simulated Data
+For each degree $i$ of relationships we computed precision and recall metrics:
 
-Pedigree simulation is performed on European populations from 1KG using the `pedsim` package.  
-Pedsim can use sex-specific genetic maps and randomly assigns the sex of each parent (or uses user-specified sexes) when using such maps.  
-Sex-specific map is preferable because men and women have different recombination rates.  
-Founders for the pedigree simulation are selected from 1000genomes HD genotype chip data, CEU population.  
-CEU data consists of trios, and we select no more than one member of each trio as founder.  
+$$\mathrm{Precision}(i) = \frac{\mathrm{TP}(i)}{\mathrm{TP}(i) + \mathrm{FP}(i)}; \quad \mathrm{Recall}(i) = \frac{\mathrm{TP}(i)}{\mathrm{TP}(i) + \mathrm{FN}(i)}.$$
 
-Visualization of structure of simulated pedigree is given below:
+Here $\mathrm{TP}(i)$, $\mathrm{FP}(i)$, $\mathrm{FN}(i)$ are the numbers of true positive, false positive, and false negative relationship matches predicted for the degree $i$.
+In our analysis we used non-exact (fuzzy) interval metrics.
+For the 1st degree, we require an exact match.
+For the 2nd, 3rd, and 4th degrees, we allow a degree interval of $\pm 1$.
+For example, for the 2nd true degree we consider a predicted 3rd degree as a true positive match.
+For the 5th+ degrees, we use the ERSA confidence intervals which are typically 3-4 degrees wide.
+For 10th+ degrees, these intervals are 6-7 degrees wide.
 
-![image](https://user-images.githubusercontent.com/19895289/123947763-6a3cc500-d9a9-11eb-8cf9-8bd8d47e6e33.png)
+<p align="center">
+    <img src="./precision & recall.png" alt="drawing"/>
+</p>
 
-#### How to run simulation
+## Evaluation on the Hapmap Data
 
-Use command simulate. Options --input and --samples are not needed in this case.
+Use command `hapmap` for the downloading and preparation of the Hapmap CEU data.
```

Comment:
hapmap does not download hapmap CEU data. It uses CEU genomes from 1000 Genomes dataset that should be already downloaded.

Reply:
Hapmap was removed completely from GRAME.